elyra/pipeline/tests/test_handlers.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import json
+import pytest
+import tornado
+
+from tornado.httpclient import HTTPClientError
+
+try:
+    import importlib.resources as pkg_resources
+except ImportError:
+    # Try backported to PY<37 `importlib_resources`.
+    import importlib_resources as pkg_resources
+
+from .. import resources
+
+
+# Set Elyra server extension as enabled (overriding server_config fixture from jupyter_server)
+@pytest.fixture
+def jp_server_config():
```

Comment:
We should move this definition (as well as the one in the [metadata tests](https://github.com/elyra-ai/elyra/blob/7f7a1cf006232f27a6f41f252d97113d7b8b39dd/elyra/metadata/tests/conftest.py#L89-L98)) to the [parent conftest.py](https://github.com/elyra-ai/elyra/blob/master/conftest.py) since that will become the location for fixture definitions that span all subsystems.  The contents stuff will need that as well.

Note: I'm not certain if this location should be more in the elyra location, rather than the root of the repo. Will have to see how CI goes, etc.

Reply:
had the same thought, but not sure where the location should be.

Reply:
Yeah.  I don't know pytest well enough to know if added to the root of the repo, then would all tests need to run from there or not.  Also, we should probably decide if we want users to be able to run all tests after installation - I think its a good idea - at least server unit tests.  It might make more sense to move the top-most file into elyra.

Regardless, we should remove the duplication with this PR, whether that means moving the top-most file down or creating another in elyra.  The contents/properties handler tests will need this as well.

Reply:
Created #1578 for agreement on how to best refactor backend tests aiming for better code reuse.